scientific/src/commonTest/kotlin/unit/TimeUnitTest.kt

```diff
@@ -163,4 +164,27 @@ class TimeUnitTest {
     fun timeFromWeightAndMassFlowRateTest() {
         assertEqualScientificValue(1(Hour), 2(Gram) / 2(Gram per Hour))
     }
+
+    @Test
+    fun timeFromKinematicViscosityAndAreaTest() {
+        assertEqualScientificValue(1(Second), (2(SquareMeter) / 2(SquareMeter per Second)))
+        assertEqualScientificValue(
```

Comment:
Also since derived units/quantities can be difficult to grasp at first I believe it would be better to split that into multiple test functions each one with just one assert and descriptive name (see my previous comment)

It's not a deal breaker but I assure you whoever will wander into this piece of code in a months time will bless you for doing that ;)

Reply:
or at least add comments before asserts stating differences between those asserts
`//Time from Imperial Area by Metric Kinematic Viscosity`

Reply:
Very good point, i will proceed with the smaller tests recommendation.